.github/workflows/golang.yml

```diff
@@ -40,12 +40,16 @@ jobs:
       matrix:
         FABRIC_VERSION:
           - '2.5.3'
+          - '3.0.0-preview'
         CREATE_CHANNEL:
           - create_channel
           - existing_channel
+        CONSENSUS:
+          - BFT
+          - RAFT
```

Comment:
Since not all of the combinations are valid, it might be easier to explicitly specify all of the combinations to be run, as in this workflow:

https://github.com/hyperledger/fabric-gateway-java/blob/0d4253f18f3540be1a2bf6a74f8bda110a766923/.github/workflows/test.yml#L44-L51

Alternatively you could use a guard something like this on the step you don't want run:
```yaml
if: !(matrix.FABRIC_VERSION == '2.5.3' && matrix.CONSENSUS == 'BFT')
```